pkg/app/execontext.go

```diff
@@ -49,26 +50,28 @@ func (ref *ExecutionContext) FailOn(err error) {
 	errutil.FailOn(err)
 }
 
-func exit(exitCode int) {
-	ShowCommunityInfo()
+func exit(exitCode int, jsonFlag string) {
```

Comment:
should make `exit` a methon in *ExecutionContext. that way there'll be no need to pass ref.Out.JSONFlag an a parameter.

Reply:
Perfect, done, thanks a lot!